docs/src/api/class-request.md

```diff
@@ -221,3 +221,14 @@ print(request.timing)
 - returns: <[string]>
 
 URL of the request.
+
+## method: Request.getPayloadAsJson
+* langs: csharp
+- returns: <[JsonDocument]>
+
+Returns the <see cref="JsonDocument"/> representation of <see cref="PostDataBuffer"/>.
```

Comment:
can we stick with md format in the docs and use [JsonDocument] here instead of a formatted tag? our docs generator already has support for link substitution and you probably have it in the xml doc gen too

Reply:
Since `JsonDocument` is a .NET type, I think this will break the documentation stuff, because the type won't be resolved... My thinking was that maybe I can define the types separately for C# only, but that felt like an obscure way of doing this, considering this is .NET only.. I'm torn :-).

Reply:
We have language-specific link maps(see e.g. [this one](https://github.com/microsoft/playwright.dev/blob/80bf24fcfe11d2773e9802b5817cf8fbc690b49a/common/links-java.md#L15) which is java-specific), so it should not break anything. I'm not saying that this is necessarily better than what you are doing in this PR but I believe it's easier to stick to one and the same approach across all languages.

Reply:
I did not know that! Perfect, I'll use that!